.github/pull_request_template.md

```diff
@@ -1,3 +1,34 @@
+## For Internal Contributors
+
+* Follow trunk workflow(https://trunkbaseddevelopment.com/)
```

Comment:
For normal situations, lets follow trunk workflow..
But if there is a major change - say, rewrite of SDK or major version update such as v4 -> v5..
We can choose other flows